examples/reference/elements/matplotlib/RGB.ipynb

```diff
@@ -28,7 +28,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The ``RGB`` element is a subclass of [``Image``](./Image.ipynb) that supports red, green, blue channels. One of the simplest ways of creating an ``RGB`` element is to load an image file (such as PNG) off disk, using the ``load_image`` classmethod:"
+    "``RGB`` represents a regularly spaced 2D grid of an underlying continuous space of RGB(A) (red, green, blue and alpha) color space values. The definition of the grid closely matches the semantics of an Image and in the simplest case the grid may be specified as a ``NxMx3`` or ``NxMx4`` array of values along with a bounds, but it may also be defined through explicit and regularly space x/y-coordinate arrays. The two most basic supported constructors of an ``RGB`` element therefore include:\n",
```

Comment:
```suggestion
    "``RGB`` represents a regularly spaced 2D grid of an underlying continuous space of RGB(A) (red, green, blue and alpha) color space values. The definition of the grid closely matches the semantics of an Image and in the simplest case the grid may be specified as a ``NxMx3`` or ``NxMx4`` array of values along with a bounds, but it may also be defined through explicit and regularly spaced x/y-coordinate arrays. The two most basic supported constructors of an ``RGB`` element therefore include:\n",
```